source/common/res/features/pacing/settings.json

```diff
@@ -8,7 +8,8 @@
       "options": [
                     { "name": "Disabled", "value": "0" },
                     { "name": "Show Full Amount", "value": "1" },
-                    { "name": "Show Simple Indicator", "value": "2" }                    
+                    { "name": "Show Days Ahead/Behind Schedule", "value": "2" },
```

Comment:
Since value 2 already exists, shouldn't it be left alone and the new option added to the end of the list? If not, users that currently have Show Simple Indicator as their choice will now get the new option. Probably not what they expect.